### PR TITLE
Add signal to SceneView Widget

### PR DESCRIPTION
**Pull request by jennyfothergill**
## Description

I added a signal that is emitted when the widget starts rendering. This is useful to me for connecting it to another widget.

Resolves: https://github.com/cmelab/GIXStapose/issues/5

## Change log

```

```

## Checklist:
- [x] I have reviewed the [**Contributor Guidelines**](https://github.com/glotzerlab/fresnel/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Fresnel Contributor Agreement**](https://github.com/glotzerlab/fresnel/blob/master/ContributorAgreement.md).
- [x] My name is on the [list of authors](https://github.com/glotzerlab/fresnel/blob/master/doc/credits.rst).

### CODE REVIEW

File: fresnel/interact.py [hunk 1]
```diff
@@ -185,6 +185,8 @@ class SceneView(QWidget):
         - :doc:`examples/02-Advanced-topics/03-Interactive-scene-view`
     """
 
+    update_signal = QtCore.Signal(camera.Camera)
```
Comment (joaander) [inline fresnel/interact.py:188]: Good to see this working. Can we rename the signal to `rendering`?

Comment (jennyfothergill) [reply fresnel/interact.py]: of course!

File: fresnel/interact.py [hunk 2]
```diff
@@ -277,6 +279,9 @@ def _stop_rendering(self):
         self.rendering = False
 
     def _start_rendering(self):
+        # send signal
+        self.update_signal.emit(self._scene.camera)
```
Comment (joaander) [inline fresnel/interact.py:283]:
```suggestion
        self.rendering.emit(self._scene.camera)
```

**Comment by joaander** [comment]
I am going to merge this and fixup the error on master.

File: fresnel/interact.py (hunk 2 above)
Comment (jennyfothergill) [reply fresnel/interact.py:283]: oh shoot! I'm sorry I didn't see this. I get flooded with a lot of github notifications that I'm not always sure if they're intended for me or a broader group I'm part of... Anyway, thanks for merging it!

**Pull request merged**